ontology.ttl

```diff
@@ -864,11 +864,11 @@ oe:dos_handeltOver a owl:ObjectProperty;
     vs:term_status "testing";
     rdfs:domain oe:Dossier;
     rdfs:range [
-        owl:unionOf (oe:Aanduidingsobject oe:Erfgoedobject)
+        owl:unionOf (oe:Aanduidingsobject oe:Erfgoedobject oe:Plan oe:Besluit)
```

Comment:
Een oe:Plan is specifiek een beheersplan of een erfgoedrichtplan, niet gewoon een plannetje of situatieschets. Klopt dat met hoe je het wil gebruiken? Ik kan me er iets bij voorstellen, tenminste als het om beheersplannen gaat.

En wanneer gaat een dossier over een Besluit? Een besluit is normaal iets dat een beslissing vastlegt. In de praktijk leggen we daarin aanduidingsobjecten vast of schaffen ze af. Maar ik zie niet meteen wat de link is met toelatingen?

Reply:
Het gaat inderdaad over "onze" plannen. Ik ben eigenlijk ook niet volledig tevreden met deze oplossing. De afbakening van een toelatingsdossier bevat potentieel meerdere objecten. Elk object is gelinkt aan potentieel meerdere besluiten. De toelating zelf gaat echter steeds in op maar 1 van deze objecten. De business wou wel dat de andere overlappende objecten, en bijhorende besluiten, ook in het dossier voorkwamen. Dat hebben we nu opgelost door te kiezen voor 1 primair object (oe:dos_handeltOver) en de rest gewoon te koppelen met een relation. Idem voor de besluiten. Moest er maar 1 besluit zijn per object is dit eigenlijk niet nodig. Hetzelfde verhaal voor de plannen. Informatie uit het besluit, het plan en het object is nodig in de brieven. Dus ofwel overtuigen we de business om te gaan naar slechts 1 van elk per dossier. Of bvb. nog meerdere objecten maar met steeds maar 1 besluit per object.

ZIe ook https://doc.onroerenderfgoed.be/docs/toelatingen_voor_handelingen/02_proces_toelatingen_handelingen.html#tl1-5-tabblad-onroerend-erfgoed

Reply:
Ok, daar moet ik even op inlezen en over nadenken

Reply:
Heb wat zitten zoeken, maar ik kan niet meteen een echt voorbeeld van een aanduidingsobject met meerdere besluiten vinden. Heb jij er? Dat moet een vrij uitzonderlijk iets zijn. Normaal zul je bij een aanduidingsobject van het type bescherming (ik neem aan dat vaststellingen hier niet meespelen, erfgoedlandschappen en Unesco kan ik niet zo goed inschatten) bijna altijd exact 1 (voorlopig of definitief) beschermingsbesluit hebben (opgelet, dat zijn andere aanduidingsobjecten). Soms ga je een vreemde situatie hebben met een aanduidingsobject dat ontstaan is na een gedeeltelijke vernietiging van een bescherming, dan heb je denk ik een koppeling met het oorspronkelijke beschermingsbesluit en een koppeling met het wijzigingsbesluit. Lijkt me alvast goed om eens op te lijsten wat de combinaties zijn.

Ik neem aan dat je nu 1 object met 1 besluit wil verplichten om te kunnen komen tot zinnen zoals "beschermd als blablabla volgens het besluit van X, gewijzigd op Y"? Hebben we al een overzicht van alle mogelijke combinaties? Ik vraag me af of we de relatie object <-> besluit niet duidelijker moeten maken bij het aanduidingobject zelf. Vanuit je toelating zou je door een koppeling met een aanduidingsobject over alle nodige info moeten beschikken. Dat besluit nog eens extra bijhouden is veel overhead voor iets wat zelden voorkomt.

Het onderscheid tussen een primair object en secundaire objecten is voor mij prima, maar ik zie daarvoor liefst altijd specifiekere relaties dan "relation". Op dit moment lijkt oe:dos_handeltOver eerder over alle gekoppelde objecten te gaan en heb je nog een extra relatie nodig voor de voorkeursobjecten. In de mokups heet dit "relevant voor dit dossier", dat lijkt aan te geven dat het er toch meerdere zouden kunnen zijn? Ik zou naast oe:dos_handeltOver een tweede relatie oe:dos_handeltPrimairOver invoeren (je zou ook het oorspronkelijke oe:dos_heeftAlsOnderwerp kunnen hergebruiken, maar dat klinkt dan weer als de titel van het dosser).

Ik ben nog altijd niet goed mee met de plannen. In de analyse zie ik die ook niet staan. Geven we toelatingen op dingen die bepaald zijn in een beheersplan, maar niet aangeduid zijn? Of hoe moet ik dat zien?

Reply:
Dat deel van die plannen staat hier: https://doc.onroerenderfgoed.be/docs/toelatingen_voor_handelingen/02_proces_toelatingen_handelingen.html#tl1-6-tabblad-beheersplannen

Het komt erop neer dat er ook informatie uit maximaal 1 plan in die brieven staat. 

Voor de erfgoedobjecten zullen we dan oe:dos_handeltPrimairOver en oe:dos_handeltOver gebruiken.

Voor de besluiten moeten we denk ik een oplossing voorzien. Zelf al zijn er maar enkele aanduidingsobjecten met meerdere besluiten, vroeg of laat komen die voor een toelating binnen en als we dan niets voorzien hebben loopt het systeem vast. Hetzelfde met de plannen. Dus de opties daar blijven:

- gebruiker verwijdert de niet relevante besluiten en plannen uit de lijst zodat er maar 1 overblijft (daar was de business niet echt voor gewonnen al weet ik ook niet waarom)
- gebruiker markeert 1 van de plannen of besluiten als relevant en de rest als gewoon gekoppeld, hier hebben we dan nog 2 relatietypes nodig

Reply:
Net gemerkt dat er al een relatie oe:aanduid_werdBeslistIn is die kan gebruikt worden om aan te geven waar de rechtsgevolgen van een bepaald aanduidingsobject uit voortvloeien. Dus, je hebt dan de ketting oe:Dossier oe:handeltPrimairOver oe:Aanduidingsobject oe:aanduid_werdBeslistIn oe:Besluit.

Kan je die gebruiken?

Reply:
Dat ga ik eens bekijken, wat doen we dan nog met die plannen?

Reply:
Voor de besluiten is het ok, de andere kunnen we dan koppelen met oe:besluit_handeltOver.
Als we nog iets vinden voor de plannen kan deze dicht.

Reply:
De plannen heb ik gelaten bij oe:dos_handeltOver, dus die kan je daar gewoon gebruiken. Ideaal is het misschien niet, maar weer een andere relatie is dat ook niet.